Scripts/Dialogue/DataStructures/3 - IntroToQueues.txt

```diff
@@ -1,22 +1,25 @@
-Explain Queues & how the inventory is green + has text as to what it is
+You might have noticed that my inventory has changed modes.
+Now it is in "Queue" mode, which is indicated by it being green and having the words "Queue" above it.
+In Queue mode, my inventory acts like the data structure, queue. Let me explain more about queues.
```

Comment:
"In Queue mode, my inventory acts like the data structure, queue." This is real clunky as a sentence, reword it

Reply:
is it cause I use queue twice in the sentence? Would "In this mode, my inventory acts like the data structure, queue" be better? Or "In this mode, my inventory acts like a queue data structure"?

Reply:
"In this mode, my inventory acts like a queue data structure" I like this one